models/marts/performance/fct_chained_views_dependencies.sql

```diff
@@ -0,0 +1,21 @@
+with all_relationships as (
+    select  
+        *
+    from {{ ref('int_all_dag_relationships') }}
+    where distance <> 0
+),
+
+final as (
+    select
+        parent,
+        child, -- the model with potentially long run time / compilation time, improve performance by breaking the upstream chain of views
+        distance,
+        path,
+        is_dependent_on_chain_of_views
```

Comment:
We could remove this column from the `fct` no? (we are already filtering on it)

Reply:
sure, i'll remove it